src/dss/utils.py

```diff
@@ -81,3 +113,79 @@ def get_lightkube_client(kubeconfig: Optional[str] = None):
     kubeconfig = KubeConfig.from_file(kubeconfig)
     lightkube_client = Client(config=kubeconfig)
     return lightkube_client
+
+
+def get_mlflow_tracking_uri() -> str:
+    return f"http://{MLFLOW_DEPLOYMENT_NAME}.{DSS_NAMESPACE}.svc.cluster.local:5000"
+
+
+def get_notebook_url(name: str, namespace: str, lightkube_client: Client) -> str:
+    """
+    Returns the URL of the notebook server given the name of the server.
+
+    Assumes the following conventions:
+    * the notebook server is exposed by a service of the same name
+    * the notebook service exposes the notebook on the first port in the service
+    """
+    service = lightkube_client.get(Service, namespace=DSS_NAMESPACE, name=name)
```

Comment:
We are not handling case when this line  fails we are not catching it here nor we do it in the function where we use it.

Reply:
Added catching inside the function and taking that into consideration in the function above, lmk what you think.